CMakeLists.txt

```diff
@@ -776,6 +776,7 @@ if(WIN32)
     add_definitions(-DFEATURE_COMINTEROP_APARTMENT_SUPPORT)
     add_definitions(-DFEATURE_COMINTEROP_UNMANAGED_ACTIVATION)
     add_definitions(-DFEATURE_COMINTEROP_WINRT_MANAGED_ACTIVATION)
+    add_definitions(-DFEATURE_DELAYLOAD_WINRT)
```

Comment:
Using /DelayLoad (https://msdn.microsoft.com/en-us/library/yx9zd12s.aspx), we could specify the libraries to be delay loaded. However, it will require us to continue to link against RuntimeObject.lib that, in its current implementation, imports the functions from the libs we want to delay load. 

However, in a future toolset update, the import libs could change (e.g. APIs could move from one apiset to another), and thus, while we will delay load the older libs, we would have imported functions from the new one, and thus, break CoreCLR load on downlevel again.